firstpartydata/tests/applyfpd/invalid-fpd-all-bidder-app.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
Just curious, what is the `-all-` portion of these test names referring to?

I was having a little trouble understanding what the test case was from the name. Maybe change these to something like: 
`invalid-fpd-req-root-app-not-defined.json`
`invalid-fpd-req-root-site-not-defined.json`
`invalid-fpd-req-root-user-not-defined.json`
`invalid-fpd-cant-set-site-page-empty.json`

Reply:
`-all-` all bidder configs in request. Will rename them.

Reply:
Renamed